cmd/ddev/cmd/import.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"log"
+
+	"os"
+
+	"github.com/drud/ddev/pkg/dockerutil"
+	"github.com/drud/ddev/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+// ImportCmd represents the `ddev import` command.
+var ImportCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Import files and database using a configured provider plugin.",
+	Long:  ``,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		if len(args) > 0 {
+			err := cmd.Usage()
+			util.CheckErr(err)
+			os.Exit(0)
+		}
+
+		client := dockerutil.GetDockerClient()
+
+		err := dockerutil.EnsureNetwork(client, netName)
```

Comment:
This is repeated all over the place, my next commit refactors into a single `dockerNetworkPreRun()` function for consistency. It also changes the error reporting to use `util.Failed()`